### PR TITLE
login page modification and logout button for home page

### PR DESCRIPTION
**Pull request by hzaw28**

### CODE REVIEW

File: src/views/Login.jsx [hunk 1]
```diff
@@ -55,27 +56,33 @@ class Login extends Component {
     } = this.state;
 
     return (
-      <div>
+      <div id="login-form">
         <h2>Login</h2>
+        <br/>
         <label htmlFor="email">Email</label>
```
Comment (solkaz) [inline src/views/Login.jsx]: You can set the label within the `TextField` component. I'd remove this

**Review by solkaz — CHANGES_REQUESTED** [review]

File: src/views/Login.jsx (hunk 1 above)
Comment (heinhtetzaw95) [inline src/views/Login.jsx]: Sorry, I don't get what you mean. <br/> is for new line?

Comment (heinhtetzaw95) [inline src/views/Login.jsx]: fixed

Comment (hzaw28) [reply src/views/Login.jsx]: `&nbsp`s removed

Comment (solkaz) [reply src/views/Login.jsx]: I meant the label element. The `TextField` component has floating label props: http://www.material-ui.com/#/components/text-field

Comment (hzaw28) [reply src/views/Login.jsx]: Oh I see, thanks.

File: src/views/Login.jsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ class Login extends Component {
     } = this.state;
 
     return (
-      <div id="login-form">
```
Comment (solkaz) [inline src/views/Login.jsx]: This needs to be a div, not a form.

File: src/views/Login.jsx [hunk 3]
```diff
@@ -77,11 +77,11 @@ class Login extends Component {
         />
         <br/>
         <RaisedButton
-          onClick={this.onSubmit}
+          onSubmit={this.handleOnSubmit}
```
Comment (solkaz) [inline src/views/Login.jsx]: Again, this shouldn't be a form. For a raised button, you would want to use the `onTouchTap` event (instead of `onClick`)

**Review by solkaz — CHANGES_REQUESTED** [review]

File: src/views/Login.jsx (hunk 2 above)
Comment (hzaw28) [reply src/views/Login.jsx]: yup

File: src/views/Login.jsx [hunk 4]
```diff
@@ -77,11 +77,11 @@ class Login extends Component {
         />
         <br/>
         <RaisedButton
-          onSubmit={this.handleOnSubmit}
+          onClick={this.onSubmit}
```
Comment (solkaz) [inline src/views/Login.jsx]: this should be `onTouchTap`. http://www.material-ui.com/#/components/raised-button

**Review by solkaz — CHANGES_REQUESTED** [review]

File: src/views/Login.jsx (hunk 4 above)
Comment (hzaw28) [reply src/views/Login.jsx]: Well, onClick works too, but okay. onTouchTap. There it is.

**Comment by solkaz** [comment]
Changes look good. However, since I've already merged my branch, you should make a new pull request to the master branch

**Pull request closed without merging**